source/manual/get-started.html.md

```diff
@@ -105,6 +108,41 @@ Some processes are dependent on your local gov.uk setup being in `~/govuk`:
     mac$ cd ~/govuk
     mac$ git clone git@github.com:alphagov/govuk-puppet.git
 
+### 3.1 Ensure your Ruby setup works
+
+You'll want to install the [specific version of Ruby in govuk-puppet][https://github.com/alphagov/govuk-puppet/blob/master/.ruby-version#L1] (currently `rbenv install 1.9.3-p550`), as the default system Ruby is too locked-down.
+
+Now verify that rbenv is working as expected:
+
+    mac$ cd ~/govuk/govuk-puppet
+    mac$ rbenv versions
+    
+This should output something like:
```

Comment:
I wonder if this is a bit of repetition from the rbenv docs?

Reply:
Getting a working rbenv setup is surprisingly fiddly (someone on Slack mentioned they've never managed to get theirs working!) so I think it's useful to provide sanity checks to confirm it is set up as expected.

Agreed that this makes the whole page a bit Ruby-centric though - is it worth splitting out into a separate page I wonder?

Reply:
No let's keep it all in one place. The next person to do this can refine it if they wish :)